frontend/src/lib/error/index.ts

```diff
@@ -60,6 +72,8 @@ function setupGlobalErrorHandlers(error: Writable<App.Error | null>): void {
       return;
     }
 
+    // Try our best to only show the user errors that are from our code
+    const showToUser = !!event.reason && !errorIsFromExtension(event.reason as Error);
```

Comment:
should include the sendBeacon check below on line 94 in here and then remove that whole branch.